node_modules/oae-principals/lib/api.user.js

```diff
@@ -475,16 +699,30 @@ var updateUser = module.exports.updateUser = function(ctx, userId, profileFields
             return callback(err);
         }
 
+        // If the user updated his email address we need to verify it
+        if (profileFields.email && oldUser.email !== profileFields.email) {
+            profileFields.emailVerified = false;
```

Comment:
I assume we'd do something similar like T&C, where the UI demands they fill it in before proceeding?